timemachine/fe/free_energy.py

```diff
@@ -404,7 +409,10 @@ def run_production_local_steps(n_steps: int) -> Tuple[NDArray, NDArray]:
             )
             coords.append(x_t)
             boxes.append(box_t)
-        return np.concatenate(coords), np.concatenate(boxes)
+
+        final_velocities = ctxt.get_v_t()
+
```

Comment:
same sanity check, does ctxt.get_x_t() match coords[-1]? i.e. making sure that the last frame of coords are in sync with the velocities.

Reply:
Assertion added in [d151e39](https://github.com/proteneer/timemachine/pull/1142/commits/d151e39490322bf88df7f07989cad682fa6fa785)